cores/arduino/WString.h

```diff
@@ -50,6 +50,9 @@ class String
 	typedef void (String::*StringIfHelperType)() const;
 	void StringIfHelper() const {}
 
+	static size_t const FLT_MAX_DECIMAL_PLACES = 10;
```

Comment:
I would like to propose to use `FLT_DECIMAL_DIG` and `DBL_DECIMAL_DIG` instead of harding both to 10 that is somewhere in the middle of the real possible values (7 and 16). Or if we wanto to stay C99 `DECIMAL_DIG`.